Test/Unit/Cron/DeactivateQuoteTest.php

```diff
@@ -19,113 +19,52 @@
-        );
-
-        $this->deactivateQuote = $this->getMockBuilder(DeactivateQuote::class)
-            ->setConstructorArgs(
-                [
-                    $this->resourceConnection,
-                    $this->bugsnag
-                ]
-            )
-            ->enableProxyingToOriginalMethods()
-            ->getMock();
-
-        $this->connection = $this->createPartialMock(Mysql::class, ['query']);
+        if (!class_exists('\Magento\TestFramework\Helper\Bootstrap')) {
+            return;
+        }
+        $this->objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
+        $this->deactivateQuote = $this->objectManager->create(DeactivateQuote::class);
     }
 
     /**
      * @test
-     * @param $data
-     * @dataProvider execute_dataProvider
      */
-    public function execute($data)
+    public function execute()
     {
-        $this->resourceConnection->expects(self::once())->method('getConnection')->willReturn($this->connection);
-
-        $this->resourceConnection->method('getTableName')->withConsecutive(
-            ['quote'],
-            ['sales_order'],
-            ['sales_order_payment']
-        )->willReturnOnConsecutiveCalls('quote', 'sales_order', 'sales_order_payment');
-
-        $this->connection->expects(self::once())->method('query')->will($this->returnCallback(
-            function ($data) {
-                $this->assertEquals(
-                    'UPDATE quote SET is_active = 0 WHERE is_active = 1 AND entity_id IN (SELECT so.quote_id FROM sales_order AS so INNER JOIN sales_order_payment AS sop ON so.entity_id = sop.parent_id WHERE sop.method = "boltpay")',
-                    $data
-                );
-            }
-        ));
+        $quote = TestUtils::createQuote(['is_active' => true]);
+        $quoteId = $quote->getId();
+        $payment = $this->objectManager->create(Payment::class);
+        $payment->setMethod(\Bolt\Boltpay\Model\Payment::METHOD_CODE);
+
+        $paymentData = [
+            'transaction_reference' => 'XXXXX',
+            'transaction_state' => 'cc_payment:pending',
+        ];
+        $payment->setAdditionalInformation(array_merge((array)$payment->getAdditionalInformation(), $paymentData));
 
-        if ($data['exception']) {
-            $this->connection->expects(self::once())->method('query')->willThrowException(new \Exception($data['exception']));
-            $this->bugsnag->expects(self::once())->method('notifyException')->will(
-                $this->returnCallback(
-                    function ($data) {
-                        $this->assertEquals(__('Error when run query'), $data->getMessage());
-                    }
-                )
-            );
-        } else {
-            $this->connection->expects(self::once())->method('query')->willReturnSelf();
-        }
+        $order = TestUtils::createDumpyOrder(['quote_id' => $quoteId], [], [], Order::STATE_PENDING_PAYMENT, Order::STATE_PENDING_PAYMENT, $payment);
```

Comment:
Is this method actually called `createDumpyOrder`? That doesn't seem right but M2 is full of surprises :D

Reply:
@johnmccombs1 bumping this

Reply:
@njfaries Yes, it calls the createDumpyOrder method. Can you please let me know what is your concern here?

Reply:
@johnmccombs1 it looks like a typo is my concern, I'm not sure how I didn't notice before in prior PRs. Is the intent of the method not to create a dummy order?

Reply:
@nirajjayant Can you review and approve the PR so I can merge it?

Reply:
@johnmccombs1 - do you need me or @njfaries to review?

Reply:
nirajjayant Sorry, my typo. I mean @njfaries